webapp/src/components/EstateCard/EstateCard.css

```diff
@@ -1,3 +1,4 @@
+/* TODO: width rules are duplicated on ParcelCard */
```

Comment:
What do you think if we create an issue to create a `assetCard.css` where we can place all the CSS rules related to cards?